internal/kibana/test/httptest.go

```diff
@@ -54,11 +57,11 @@ func pathForURL(url string) string {
 }
 
 func recordRequest(t *testing.T, r *http.Request, path string) {
-	client, err := NewClient()
+	client, err := stack.NewKibanaClient()
 	require.NoError(t, err)
 
 	t.Logf("Recording %s in %s", r.URL.RequestURI(), path)
-	status, respBody, err := client.get(r.URL.RequestURI())
+	status, respBody, err := client.SendRequest(http.MethodGet, r.URL.RequestURI(), nil)
```

Comment:
Why not doing public get, post, put and delete methods?

Reply:
I don't have a strong preference, I found better to expose the most generic one. Would you prefer to expose the ones per method?